### PR TITLE
Make gitea serv use api/internal

### PR DESCRIPTION
**Pull request by sapk**
In order to limit concurrent access to database (mostly for SQLite)

Related: #4848 #2040

### CODE REVIEW

**Comment by lafriks** [comment]
Git clone tests seems to fail now

File: modules/private/key.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```
Comment (lunny) [inline modules/private/key.go]: 2018

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by sapk** [comment]
I have not much time until 2 weeks. I will look at tests since I test clone manually and it worked but I must have forget one corner case.

**Comment by sapk** [comment]
I found the stupid mistake I made. :expressionless:
This PR should be good.

**Review by lunny — APPROVED** [review]

**Comment by sapk** [comment]
@lunny I will have to verify but hook seems to only use const in models and already use private (api/internal). 
We could drop https://github.com/go-gitea/gitea/blob/master/cmd/hook.go#L69 that shouldn't be needed anymore but that will be in another PR.

**Review by BetaCat0 — APPROVED** [review]

**Review by appleboy — APPROVED** [review]
